build.gradle

```diff
@@ -148,6 +154,35 @@ signing {
     sign configurations.jars
 }
 
+def projectPom = {
+    project {
+        name project.name
+        description 'The spark-ecs-s3 project makes it possible to view an ECS bucket as a Spark dataframe'
```

Comment:
I suggest a description of:
"The spark-ecs-s3 project makes it possible to work with data stored in ECS using Apache Spark."